stubs/lief/lief/ELF/__init__.pyi

```diff
@@ -0,0 +1,3115 @@
+    def has_dynamic_symbol(self, symbol_name: str) -> bool: ...
+    def has_library(self, library_name: str) -> bool: ...
+    def has_section(self, section_name: str) -> bool: ...
+    def has_section_with_offset(self, offset: int) -> bool: ...
+    def has_section_with_va(self, virtual_address: int) -> bool: ...
+    def has_static_symbol(self, symbol_name: str) -> bool: ...
+    @overload
+    def patch_pltgot(self, symbol_name: str, address: int) -> None: ...
+    @overload
+    def patch_pltgot(self, symbol, address: int) -> None: ...
+    def permute_dynamic_symbols(self, permutation: List[int]) -> None: ...
+    @overload
+    def remove(self, dynamic_entry) -> None: ...
+    @overload
+    def remove(self, tag: DYNAMIC_TAGS) -> None: ...
+    @overload
+    def remove(self, section, clear: bool = ...) -> None: ...
+    @overload
+    def remove(self, note) -> None: ...
+    @overload
+    def remove(self, type: NOTE_TYPES) -> None: ...
+    def remove_library(self, library_name: str) -> None: ...
+    def remove_static_symbol(self, arg0) -> None: ...
+    def replace(self, *args, **kwargs) -> Any: ...
+    def section_from_offset(self, *args, **kwargs) -> Any: ...
+    def section_from_virtual_address(self, *args, **kwargs) -> Any: ...
+    def segment_from_offset(self, *args, **kwargs) -> Any: ...
+    def segment_from_virtual_address(self, *args, **kwargs) -> Any: ...
+    def strip(self) -> None: ...
+    def virtual_address_to_offset(self, virtual_address: int) -> int: ...
+    def write(self, output: str) -> None: ...
+    @overload
+    def __contains__(self, arg0: SEGMENT_TYPES) -> bool: ...
+    @overload
+    def __contains__(self, arg0: DYNAMIC_TAGS) -> bool: ...
+    @overload
+    def __contains__(self, arg0: NOTE_TYPES) -> bool: ...
+    @overload
+    def __contains__(self, arg0: SECTION_TYPES) -> bool: ...
+    def __eq__(self, arg0: Binary) -> bool: ...
+    def __getitem__(self, index) -> Any: ...
+    def __hash__(self) -> int: ...
+    @overload
+    def __iadd__(self, arg0) -> Binary: ...
+    @overload
+    def __iadd__(self, arg0) -> Binary: ...
+    @overload
+    def __iadd__(self, arg0) -> Binary: ...
+    @overload
+    def __iadd__(self, arg0) -> Binary: ...
+    @overload
+    def __isub__(self, arg0) -> Binary: ...
+    @overload
+    def __isub__(self, arg0: DYNAMIC_TAGS) -> Binary: ...
+    @overload
+    def __isub__(self, arg0) -> Binary: ...
+    @overload
+    def __isub__(self, arg0: NOTE_TYPES) -> Binary: ...
+    def __ne__(self, arg0: Binary) -> bool: ...
+    @property
+    def dtor_functions(self) -> List[lief.Function]: ...
+    @property
+    def dynamic_entries(self) -> Binary.it_dynamic_entries: ...
+    @property
+    def dynamic_relocations(self) -> Binary.it_filter_relocation: ...
+    @property
+    def dynamic_symbols(self) -> Binary.it_symbols: ...
+    @property
+    def eof_offset(self) -> Return the last offset used by the ELF binary according to both: ...
```

Comment:
I would suggest to either find out the correct type here or just leave it unannotated:

```suggestion
    def eof_offset(self): ...
```